atc/db/migration/migrations/1626194317_add_pipeline_pauses_table.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE pipeline_pauses (
+  pipeline_id integer PRIMARY KEY,
+  paused boolean NOT NULL DEFAULT TRUE,
+  paused_by text DEFAULT NULL,
+  paused_at timestamptz NOT NULL DEFAULT now()
+);
+
+ALTER TABLE pipeline_pauses
+  ADD CONSTRAINT pipeline_pauses_id_fkey FOREIGN KEY (pipeline_id) REFERENCES pipelines (id) ON DELETE CASCADE;
+
+INSERT INTO pipeline_pauses
+SELECT
+  id,
+  TRUE
+FROM
+  pipelines
+WHERE
+  pipelines.paused = TRUE;
+
+ALTER TABLE pipelines
+  DROP COLUMN paused;
```

Comment:
We would want to keep this column around, so we don't need to JOIN with the new table everytime (specially when we don't need the extra information that table provides), that way we can avoid the JOIN limit of 8 that postgres has.